lib/clustering/mean_shift.h

```diff
@@ -9,19 +9,39 @@
 #include "clustering.h"
 #include <vector>
 #include "generic/cluster.h"
+#include <Eigen/Core>
 
 namespace MouseTrack {
 
 class MeanShift: public Clustering {
 public:
 
   /// k is the number of desired clusters
-  MeanShift(const int k);
+  MeanShift(double window_size);
   /// Splits a point cloud into k clusters randomly
   std::vector<Cluster> operator()(const PointCloud& cloud) const;
 
 private:
-    const int _k;
+  // when two peaks are closer than this, they are merged.
+  const double MERGE_THRESHOLD = 0.0001;
```

Comment:
I expect, we want to tweak those parameters interactively in the future. I would prefer public accessors for them.

Reply:
will do